internal/service/alertconfiguration/resource_test.go

```diff
@@ -566,6 +570,32 @@ func TestAccConfigRSAlertConfiguration_withVictorOps(t *testing.T) {
 	})
 }
 
+func TestAccConfigRSAlertConfiguration_withSeverityOverride(t *testing.T) {
+	var (
+		projectID = acc.ProjectIDExecution(t)
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:                 func() { acc.PreCheckBasic(t) },
+		ProtoV6ProviderFactories: acc.TestAccProviderV6Factories,
+		CheckDestroy:             checkDestroy(),
+		Steps: []resource.TestStep{
+			{
+				Config: configWithSeverityOverride(projectID, "WARNING"),
+				Check: resource.ComposeAggregateTestCheckFunc(
```

Comment:
We can add an additional step modifying the value, and also capture behaviour of what happens when attribute is removed (related to Espen [thread](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/3795/files#r2459413190)).